acestep/ui/gradio/interfaces/generation_tab_secondary_controls.py

```
@@ -170,3 +170,39 @@ def build_repainting_controls() -> dict[str, Any]:
+    Returns:
+        Component map with the retake accordion and its inputs.
+    """
+
+    with gr.Accordion("Retake (variation generation)", open=False) as retake_accordion:
+        retake_variance = gr.Slider(
+            label="Retake Variance",
+            minimum=0.0,
+            maximum=1.0,
+            step=0.01,
+            value=0.0,
+            info="0=no retake (baseline). Low (0.05–0.15) = subtle variation; high (0.5+) = stronger drift.",
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)